src/main.ts

```diff
@@ -25,6 +26,7 @@ const TLAPLUS_CFG_FILE_SELECTOR: vscode.DocumentSelector = { scheme: 'file', lan
 const CHANGELOG_URL = vscode.Uri.parse('https://github.com/alygin/vscode-tlaplus/blob/master/CHANGELOG.md#change-log');
 
 const tlaDocInfos = new TlaDocumentInfos();
+export const symbolsProvider = new TlaDocumentSymbolsProvider(tlaDocInfos);
```

Comment:
While this doesn't break anything, it did end up being a change that I no longer needed; this and the change at line [88/93](https://github.com/alygin/vscode-tlaplus/pull/152/files#diff-8cfead41d88ad47d44509a8ab0a109adR93).